migrations/20240823213817_add_feedback.up.sql

```diff
@@ -0,0 +1,27 @@
+-- Add up migration script here
+
+CREATE TYPE feedback_type AS ENUM
+    ('Positive', 'Negative', 'Suggestion', 'Note');
+
+CREATE TABLE mod_feedback
+(
+    id SERIAL PRIMARY KEY NOT NULL,
+    mod_version_id INTEGER NOT NULL,
+    reviewer_id INTEGER NOT NULL,
+    feedback TEXT COLLATE pg_catalog."default" NOT NULL DEFAULT 'No feedback provided.'::text,
```

Comment:
grr why is this the default, let it be null if there's no feedback provided

Reply:
ok